client/state/selectors/get-editor-close-config.js

```diff
@@ -57,7 +57,16 @@ export default function getEditorCloseConfig( state, siteId, postType, fseParent
 	}
 
 	// Otherwise, just return to post type listings
+
+	let label = translate( 'Back' );
+	if ( postType === 'post' ) {
+		label = translate( 'View posts' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **116** times:
`translate( 'View Posts' )` *ES Score: 8*